tlspuffin/src/tls/rustls/msgs/heartbeat.rs

```diff
@@ -1,10 +1,14 @@
+use extractable_macro::Extractable;
 use puffin::codec::{Codec, Reader};
 
+use crate::protocol::TLSProtocolTypes;
 use crate::tls::rustls::msgs::base::PayloadU16;
 use crate::tls::rustls::msgs::enums::HeartbeatMessageType;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Extractable)]
+#[extractable(TLSProtocolTypes)]
 pub struct HeartbeatPayload {
+    #[extractable_ignore]
     pub typ: HeartbeatMessageType,
     pub payload: PayloadU16,
     pub fake_length: Option<u16>,
```

Comment:
I think this was ignored before.
It would be nice to limit this PR to replace previous extractations with the macro but leave more extractions to a future PR. So that evaluation will be easier in the sense that this PR should not impact the fuzzer!

Reply:
Please check this globally.